integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -100,6 +107,17 @@ spec:
               fieldRef:
                 fieldPath: status.podIP
           {{- end }}
+          {{- if eq .Values.worker.fuseEnabled true }}
+          - name: ALLUXIO_CLIENT_HOSTNAME
+            valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
```

Comment:
I'm not sure if this should be the hostIP or the podIP, pls double check if this works, especially if this will enable the process-local read on the worker pod @LuQQiu

Reply:
emmm, it should be similar to the ALLUXIO_WORKER_HOSTNAME, 
```
          env:
          - name: ALLUXIO_WORKER_HOSTNAME
            valueFrom:
              fieldRef:
                fieldPath: status.hostIP
          {{- if not .Values.worker.hostNetwork }}
          - name: ALLUXIO_WORKER_CONTAINER_HOSTNAME
            valueFrom:
              fieldRef:
                fieldPath: status.podIP
```
i saw we use hostIP for worker hostname. This works in my testing.
Not quite sure why we need a new property key `ALLUXIO_WORKER_CONTAINER_HOSTNAME` in container env?
should i create a similar `ALLUXIO_CLIENT_CONTAINER_HOSTNAME`?

Reply:
@ssz1997 If you search `alluxio.worker.container.hostname` in Alluxio property key file, you will find that property key, but there is no such a thing as `alluxio.client.container.hostname`.
Next time if you make any change, make sure you understand why the changes are needed
e.g. why we use `ALLUXIO_WORKER_CONTAINER_HSOTNAME` `alluxio.worker.container.hostname` in alluxio code base. and then think if we should add a similar `alluxio.client.container.hostname` in the alluxio java code.
Adding `ALLUXIO_CLIENT_CONTAINER_HOSTNAME` here doesn't make any sense since no application consumes this information.